dspace-api/src/main/java/org/dspace/discovery/indexobject/ItemIndexFactoryImpl.java

```diff
@@ -154,9 +154,11 @@ public SolrInputDocument buildDocument(Context context, IndexableItem indexableI
         doc.addField("latestVersion", isLatestVersion(context, item));
 
         EPerson submitter = item.getSubmitter();
-        if (submitter != null) {
-            addFacetIndex(doc, "submitter", submitter.getID().toString(),
-                    submitter.getFullName());
+        if (submitter != null && !(DSpaceServicesFactory.getInstance().getConfigurationService().getBooleanProperty(
+            "discovery.index.item.submitter.enabled", true))) {
```

Comment:
I think we should default this setting to `false`.  I'm not seeing a great reason for indexing the submitter *by default*, as it seems like that could be a privacy concern for some.

Reply:
Hi @tdonohue . Thanky for your feedback and the review (again). I've changed the default behaviour as you have suggested, now it is by default false